README.md

```diff
@@ -114,17 +114,47 @@ The provider can be built and installed locally in `$HOME` by running:
 make install-local
 ```
 
+After running `make install-local`, you will be able to reference the snapshot build
+version in any local Terraform configuration. For example:
+
+```hcl
+terraform {
+  required_providers {
+    logdna = {
+      source = "logdna.com/logdna/logdna"
+      version = "1.2.0-pre-SNAPSHOT-b9faaaa"
+    }
+  }
+}
+```
+
 ### Docker
 
 The included tooling can be used to test and build the provider inside a Docker build
 environment, without installing any dependencies locally. 
 
 You will need an ascii-armored GPG key in the root of the project at `./gpgkey.asc` for
-signing test builds.
+signing test builds. If you do not already have a personal GPG key, you can generate one
+by [following this guide](https://docs.github.com/en/github/authenticating-to-github/managing-commit-signature-verification/generating-a-new-gpg-key).
+
+Export your key to the root of this repository:
+
+```sh
+gpg --armor --export <ID> > ./gpgkey.asc
```

Comment:
The gpg key here can only be used locally for testing right? Like we'd still need the key associated to this repo in order to sign a release build

Reply:
Correct, releases will need to be generated and signed by the production key. This is only for testing the release process with `make test-release`, or other commands that build a local image first.

I'll make a note of this in the docs, then we can land this (which should then create a draft release v1.3.1, the next version for the registry)